gui/additionsPane.py

```diff
@@ -79,9 +81,12 @@ def __init__(self, parent):
         self.gangPage = GangView(self.notebook)
         self.notebook.AddPage(self.gangPage, "Fleet", tabImage = gangImg, showClose = False)
 
+        self.notes = NotesView(self.notebook)
+        self.notebook.AddPage(self.notes, "Notes", tabImage = notesImg, showClose = False)
+
         self.notebook.SetSelection(0)
 
-    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet"]
+    PANES = ["Drones", "Fighters", "Cargo", "Implants", "Boosters", "Projected", "Fleet", "Notes"]
```

Comment:
As I said it's not just git blame.  Compares are much easier down the line.

And yes, I know that you can chase the history of changes and see what folks did, but that takes more effort/time.